java/ql/src/experimental/Security/CWE/CWE-176/UnicodeBypassValidation.qhelp

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE qhelp PUBLIC "-//Semmle//qhelp//EN" "qhelp.dtd">
+<qhelp>
+    <overview>
+        <p>Security checks bypass due to a Unicode transformation</p>
```

Comment:
As the Go version of this PR, no need for a heading, and no `img` in qhelp I'm afraid

Reply:
Fixed in https://github.com/github/codeql/pull/12995/commits/dad8a03ab862a32b4187b436f478f3b2deab2a59